atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/DelegatingTransactionKeyValueService.java

```diff
@@ -95,4 +95,10 @@ public void multiPut(Map<TableReference, ? extends Map<Cell, byte[]>> valuesByTa
             throws KeyAlreadyExistsException {
         delegate.multiPut(valuesByTable, timestamp);
     }
+
+    @Override
+    public boolean isValid(long timestamp) {
```

Comment:
Atlas doesn't have "unused variable" checked active? :)

Reply:
It does not! I think the overhead to "fix" it was deemed excessive, plus some of the previous team at the time we started dealing with checkstyle exclusions were against it 😅

Reply:
We should just do it, there's actually a lot of value to this repo being the same as all the internal repos.

Reply:
+1 to adding back the check, even if takes some investment to get all checks passing. At least I'd prefix the `_timestamp` with the underscore here, so we don't have to fix when we eventually enable the check.

Reply:
I'm like 99% sure that this is an automated errorprone that can be autofixed, so not sure why it's excessive to fix.